main.py

```diff
@@ -21,10 +24,31 @@ def read_stats():
     """
     Get stats
     """
-    url = "https://api.coronatracker.com/v2/analytics/country"
+    url = "https://services8.arcgis.com/yhz7DEAMzdabE4ro/arcgis/rest/services/DZ_COVID/FeatureServer/2/query?f=json&where=1%3D1&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Report%20asc&outSR=102100&resultOffset=0&resultRecordCount=1000&cacheHint=true"
```

Comment:
can you break line and disable cache (`cacheHint`) ?

Reply:
I updated it